src/components/feedback/GoodOrBadButton.tsx

```diff
@@ -80,21 +78,21 @@ const useButtonStyle = StyleSheet.createThemeHook((theme) => ({
     marginTop: theme.spacings.xLarge,
   },
   button: {
-    backgroundColor: theme.static.background[themeColor].background,
+    backgroundColor: theme.interactive.interactive_2.default.background,
     flex: 1,
     padding: theme.spacings.xLarge,
     justifyContent: 'space-evenly',
     alignItems: 'center',
     borderWidth: theme.border.width.medium,
     borderRadius: theme.border.radius.regular,
-    borderColor: theme.static.background[themeColor].background,
+    borderColor: theme.interactive.interactive_2.default.background,
   },
   selectedButton: {
     backgroundColor: hexToRgba(
-      theme.static.background[selectedThemeColor].background,
+      theme.interactive.interactive_2.active.background,
       0.2,
     ),
```

Comment:
I don't think you need to add transparency here anymore

Reply:
Got it from here where 20% seems to be specified, but I can see that the colors are a bit lighter than what shows in Figma. Will try without transparency and see how it works :)

<img width="787" alt="image" src="https://user-images.githubusercontent.com/21310942/168237838-8b512a7b-eea4-42f8-a393-9d0d25e84a68.png">

Reply:
Hmm, that might be an outdated color or something 🤔 I think interactive_2.active should be right without transparency.